tests/software_tests/can/packet/test_first_frame.py

```
@@ -0,0 +1,612 @@
+            expected_output = data_bytes_number - ai_data_bytes_number - LONG_FF_DL_BYTES_USED
+        else:
+            expected_output = data_bytes_number - ai_data_bytes_number - SHORT_FF_DL_BYTES_USED
+        assert get_first_frame_payload_size(addressing_format=addressing_format,
+                                            dlc=dlc,
+                                            long_ff_dl_format=long_ff_dl_format) == expected_output
+        self.mock_dlc_handler.decode_dlc.assert_called_once_with(dlc)
+        self.mock_can_addressing_information.get_ai_data_bytes_number.assert_called_once_with(addressing_format)
+
+    # generate_ff_dl_bytes
+
+    @pytest.mark.parametrize("ff_dl, long_ff_dl_format, expected_ff_dl_bytes", [
```

Ruff (0.12.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)